packages/server-wallet/e2e-test/pong/controller.ts

```diff
@@ -0,0 +1,45 @@
+import {Message} from '@statechannels/wire-format';
+import {SignedState} from '@statechannels/wallet-core';
+
+import {Wallet} from '../../src/wallet';
+
+export default class PongController {
+  private readonly wallet: Wallet = new Wallet();
+
+  public async acceptMessageAndReturnReplies(message: Message): Promise<Message> {
```

Comment:
Either in this PR or shortly after, can we have this so that `pong`

1. calls `pushMessage`
2. looks at the channel result. if its turn number is at least 4, it calls `updateChannel`
3. it then responds with the new outgoing messages
?

This
A. doesn't seem like a lot of work
B. should work with the existing server-wallet
C. would add a lot of value in terms of knowing that "things are working end-to-end"
   - In particular, this is essentially the stage 0 goal!

Reply:
I'm going to do it right after, been waiting on `ChannelUpdated` being emitted by the call to `pushMessage`